### DISCUSSION TITLE
Using spell checker for the docs (perhaps other files too)

### DISCUSSION THREAD

**Discussion by juancarlosjr97** [directus/directus#4301] [Feature Requests]
I had a look at the #4299 and once I used https://www.npmjs.com/package/markdown-spellcheck, could be useful?

Honestly, I found it quite annoying initially as it was picking up many technical words, but eventually, it was good. Then I found https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker and replaced the markdown-spellcheck as I could use the plugin for any type of file. 

Perhaps is worth to use it? English is not my native language, I make mistake on a daily basis :)

**Comment by rijkvanzanten** [comment]
I actually recently started using that VS Code plugin too 🙂 

<img width="321" alt="CleanShot 2021-02-25 at 16 04 56@2x" src="https://user-images.githubusercontent.com/9141017/109216972-37788b00-7783-11eb-9b7e-dfcfb0fae7c7.png">

>  English is not my native language, I make mistake on a daily basis :)

Same! 💪🏻

**Comment by rijkvanzanten** [comment]
This has since been enabled 🙂
